.circleci/config.yml

```diff
@@ -8,43 +8,44 @@ parameters:
     type: boolean
     description: Whether the pipeline was triggered through the CircleCi API (https://circleci.com/docs/api/v2/?shell#trigger-a-new-pipeline).
     default: false
-  production:
-    type: boolean
-    description: Whether to deploy to production (integreat.app) or not (webnext.integreat-app.de).
-    default: false
+  build_config:
+    type: enum
+    description: Which build config to use and where to deploy to.
+    enum: ["integreat", "integreat-debug", "malte"]
+    default: "integreat-debug"
 
 workflows:
   version: 2
   commit: # Builds every commit. Delivers to webnext on develop.
     unless: << pipeline.parameters.api_triggered >>
     jobs:
       - build:
-          production: false
+          build_config: "integreat-debug"
       - deliver:
-          production: false
-          requires:
-            - build
           filters:
             branches:
               only:
                 - develop
+          deliver_to: "integreat-debug"
+          requires:
+            - build
 
   api-triggered-delivery: # Workflow for manual delivery
     when: << pipeline.parameters.api_triggered >>
     jobs:
       - build:
-          production: << pipeline.parameters.production >>
+          build_config: << pipeline.parameters.build_config >>
       - bump_version:
-          shouldBumpVersion: << pipeline.parameters.production >>
+          build_config: << pipeline.parameters.build_config >>
           requires:
             - build
       - deliver:
-          production: << pipeline.parameters.production >>
+          deliver_to: << pipeline.parameters.build_config >>
           requires:
             - build
             - bump_version
```

Comment:
does that mean if you manually build integreat-debug the version number is increased?

Reply:
No it shouldn't, take a look at `bump_version`. It is only bumped if config is `integreat`. Didn't find a better solution.